accname/index.html

```diff
@@ -285,8 +285,8 @@ <h2>Name and Description</h2>
         </dd>
         <dt>contents</dt>
         <dd>
-          name is generated from the Text [=nodes=] associated with the [=element=]. Although this may be allowed in addition to "author" in some <a class="termref">roles</a>, "content" is used only
-          if higher priority "author" features are not provided. Priority is defined by the <a href="#mapping_additional_nd_te">text equivalent computation</a> algorithm.
+          name is generated from the [=exclusive Text node|text node=] associated with the [=element=]. Although this may be allowed in addition to "author" in some <a class="termref">roles</a>,
```

Comment:
This changes from “the nodes” (plural) to “the text node” (singular)—is that intended? An accessible name can be generated from multiple text nodes.

For example, this URL—
```
data:text/html,<h1>Hello, <!--COMMENT-->world!</h1>
```

—plus this JavaScript—

```JS
getAccessibleName(document.querySelector("h1"))
// 'Hello, world!'

document.querySelector("h1").childNodes
// NodeList(3) [text, comment, text]
```

Reply:
Agreed "text nodes" should be plural, thank you @smockle. This does introduce inconsistency for the ReSpec linking; aliases are strongly discouraged (let alone differently aliased links) so curious if anyone disagrees with simply making it plural while in other cases we use "text node" as the link. 

Changed as part of https://github.com/w3c/aria/pull/2274/commits/bde4f90bfc312be58e09e04842d55136713a924a.